AI-and-Analytics/End-to-end-Workloads/Census/sample.json

```diff
@@ -11,7 +11,7 @@
   "ciTests": {
   	"linux": [
     {
-  		"env": ["source /opt/intel/oneapi/setvars.sh --force", "conda create -y -n aikit-modin-test -c intel -c conda-forge runipy intel-aikit-modin", "conda activate aikit-modin-test"],
+  		"env": ["source /opt/intel/oneapi/setvars.sh --force", "conda create -y -n aikit-modin --override-channels intel-aikit-modin omniscidbe4py python=3.7 runipy ray-dashboard -c intel -c conda-forge", "conda activate aikit-modin", "pip install opencensus"],
```

Comment:
@vsanghavi -- the CI system preconfigures the environment variables. I recommend that you remove the `source /opt/intel/oneapi/setvars.sh --force` step from the `env` array. If the CI system was to use `/bin/sh` (which is quite common for CI systems) the step you are performing would fail.

Also, I would expect to see the other steps in that `env` array to be in the `steps` array, so the CI system will capture any errors that arise from the `conda create` step and remaining steps. Doing so will help debug what is failing if there are problems in those steps.

Reply:
Change made to move all conda and installation steps to 'steps'